deployment/common/types/types.go

```diff
@@ -44,6 +44,8 @@ const (
 	ExecutorAccessControllerAccount  deployment.ContractType = "ExecutorAccessControllerAccount"
 	CancellerAccessControllerAccount deployment.ContractType = "CancellerAccessControllerAccount"
 	BypasserAccessControllerAccount  deployment.ContractType = "BypasserAccessControllerAccount"
+	// Data Streams contracts
+	ChannelConfigStore deployment.ContractType = "ChannelConfigStore"
```

Comment:
Should this be defined in this high level types? Or can it be kept in a data-streams type section?